omero/training-server/playbook.yml

```diff
@@ -371,6 +396,7 @@
     # or override it by defining the private variable os_system_users_password_override.
     os_system_users_password: "{{ os_system_users_password_override | default('$6$leKi5B1PgSvdA/ec$xbU3CnoSFnYdeZjEjKK5TH8SGATsW746uopssff4edpgyu.cWXGo9A.oK6wH9kIkxLCCNcORGZnnroZPMqGzN/') }}"
     apache_docker_release: "{{ apache_docker_release_override | default('0.6.0') }}"
+    omero_ms_zarr_release: "{{ omero_ms_zarr_release_override | default('latest') }}"
```

Comment:
Is this going to be tagged? If not you may want to set [`pull`](https://docs.ansible.com/ansible/latest/modules/docker_container_module.html#parameter-pull) to ensure the latest image is always pulled.

Reply:
Where would you set `pull`? (Generally, yes it will be tagged, but I assume a few more PRs before we're ready for the meeting)

Reply:
In the `docker_container` task.